scripts/us_bea/states_gdp/states_gdp.tmcf

```diff
@@ -0,0 +1,17 @@
+Node: E:states_gdp->E1
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:InflationAdjustedGDP
+observationAbout: C:states_gdp->GeoId
+observationDate: C:states_gdp->Quarter
+observationPeriod: "P3M"
+value: C:states_gdp->millions_of_chained_2012_dollars
+unit: "Millions of chained 2012 dollars"
+
+Node: E:states_gdp->E2
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:InflationAdjustedGDP
```

Comment:
A drive-by comment to sanity-check schema... where is InflationAdjustedGDP defined?

Reply:
It's defined in the MCF file in this same PR.

Reply:
I see now. That LG.